.gitmodules

```diff
@@ -1,3 +1,6 @@
 [submodule "shared-modules"]
 	path = shared-modules
 	url = https://github.com/flathub/shared-modules.git
+[submodule "minecraft-wayland"]
```

Comment:
Looks like this submodule is not being used anywhere.

Reply:
The plan seems to initially be to use the patches from there, but I rebased em and put em here locally instead. Removed it.